controllers.js

```diff
@@ -56,11 +57,28 @@ function postComments(req, res, next) {
     .catch(next);
 }
 
+function patchArticleVotes(req, res, next) {
+  const { article_id } = req.params;
+  const { inc_votes } = req.body;
+
+  if (typeof inc_votes !== "number") {
+    return res.status(400).send({ msg: "Bad request" });
+  }
+  if (isNaN(article_id)) {
+    return res.status(400).send({ msg: "Bad request" });
+  }
```

Comment:
Two issues with these `if` statements

- Sending `Error` responses should be done in your _Error Handlers_ rather than your controllers
- This logic isn't necessary, if your query takes in `inc_votes` and it's not a number, PSQL will throw an error by itself 👍 Possibly another _22P02_ but I'm not 100% on that